### DISCUSSION TITLE
Importing skin from stable making duplicates

### DISCUSSION THREAD

**Discussion by Scrollname** [ppy/osu#23420] [Ideas] [answered]
Recently I discover that if I'm trying to import 1 specific skin from stable, I'm also importing all the rest of skins that I already had in the lazer, and I'm ending up with lots of duplicated skins. I suggest that when you click import button every skin name will be checked, and if skin with same name already exist, just ignore it and don't import.

**Comment by Theighlin** [comment, marked as answer]
Have you modified said skins after importing them the first time? If so, they are technically different, and osu will reimport them again (lazer already checks for duplicated skins).

There was a bug once that duplicated skins multiple times, but has been fixed. Check you don't get any more skins when importing from stable multiple times.

**Reply by Scrollname** [reply]
You are right, it's importing only modified skins. I apologize for my stupidity.

**Reply by Theighlin** [reply]
I also got confused by this once, you're not the only one. I do wonder if there's a better way to convey this.

Maybe like the default skin is untouchable and any modification creates a modified version, when reimporting, if the name is the exact same you could append an "unmodified" or "original"? But even then I'm not sure the name is a guarantee (two different skins could have the same name, but one wouldn't be the original over the other)
